# Configuration of rrt planner
iterations: 5000
deltaX: 0.5
greedy: 0.1 